lib/improver/utilities/temporal.py

```diff
@@ -234,39 +234,22 @@ def iris_time_to_datetime(time_coord):
     return [value.point for value in coord.cells()]
 
 
-def datetime_to_iris_time(dt_in, time_units="hours"):
+def datetime_to_iris_time(dt_in, dtype=np.int64):
```

Comment:
Do we need the `dtype` argument at all?

Reply:
No, I don't think we do. We can just ensure it returns int64.